webpack.config.js

```diff
@@ -14,10 +14,18 @@ var config = {
     libraryTarget: 'umd'
   },
   plugins : [
+    {
+      apply: function apply(compiler) {
+        compiler.parser.plugin('expression global', function expressionGlobalPlugin() {
+          this.state.module.addVariable('global', "(function() { return this; }()) || Function('return this')()")
```

Comment:
I'm curious how much going to great lengths like Lodash does is buying us. Should we settle for this, or should we do a more sophisticated check like https://github.com/rackt/redux/pull/611#issuecomment-177679332?

Also, are we breaking anyone with this? Are there environments where `Object.getPrototypeOf` (as we had before) would “just work” but `Function('return this')()"` would fail? I'm thinking stuff like Firefox extensions etc.